rtv/page.py

```diff
@@ -369,11 +416,16 @@ def _add_cursor(self):
     def _remove_cursor(self):
         self._edit_cursor(curses.A_NORMAL)
 
-    def _move_cursor(self, direction):
+    def _move_cursor(self, direction, page_ud=False):
 
         self._remove_cursor()
 
-        valid, redraw = self.nav.move(direction, len(self._subwindows))
+        if page_ud:
+            valid, redraw = self.nav.move_page(direction,
+                                               len(self._subwindows))
```

Comment:
One way to avoid it is always redraw from the bottom (nav.inverted==True) when page up and always redraw from the top (nav.inverted==False) when page down. In this way I don't have to think about the length of the contents of each post.